packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesConfiguration.kt

```diff
@@ -156,6 +178,11 @@ class UpdatesConfiguration {
     }
   }
 
+  val codeSigningConfiguration: Crypto.CodeSigningConfiguration?
+    get() = codeSigningCertificate?.let {
+      Crypto.CodeSigningConfiguration(it, codeSigningMetadata)
```

Comment:
Should this be memoized? (Though we'd want to dirty the value if codeSigningMetadata were re-assigned)

Reply:
Yep. Technically this class should be an immutable data class. I'll change that in a follow-up.